igor/igor_app/urls.py

```diff
@@ -1,6 +1,10 @@
-from django.conf.urls import url
+from django.conf.urls import url, include
+from django.urls import path
 from . import views
 
 urlpatterns = [
-    url(r'^$', views.index, name="index"),
+    url(r'^$', views.profile),
+    path(r'logout', include('login_reg.urls')),
```

Comment:
this line here is what I am referring to

Reply:
The line in red? Line 5?
I may have pushed the wrong thing, there's no index line in the file I have.
igor_app/urls.py starts right from line 6 views.profile

Should I try again?

Reply:
no I was referring to line 7 path(r'logout', include('login_reg.urls')) 

That red line means you removed it from the file in your commit

Reply:
How should I go about routing logout?

Reply:
I think I got it. Just deleted the path and left include

Reply:
the way the routes work '/logout' sends you down the logout function in urls.  look in the urls.py from login_reg to see the route.  All igor routes will start with '/igor/' so that route you set up will only be seen by going to '/igor/logout'

Reply:
if you create a logout button you will just need to set the href to '/logout' and it will log the user out

Reply:
so technically the app I imported from my last project is handling that login route and we wont need to add it here